### PR TITLE
--fast flag for quick runs.

### PR DESCRIPTION
**Pull request by paulirish**
so fast now.

### CODE REVIEW

File: test/module/index.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ describe('Module Tests', function() {
 
   it('should be able to run lighthouse with just a url and options', function() {
     const lighthouse = require('../..');
-    return lighthouse(VALID_TEST_URL, {})
+    return lighthouse(VALID_TEST_URL, {fast: true})
```
Comment (paullewis) [inline test/module/index.js]: This is a breaking change. I don't think we should default it like this.

**Comment by paullewis** [comment]
I think we should take a different approach here: ideally we should take a list of tech that we want to test (default: all), and we load the appropriate gatherers and set up the scheduler to only run the required phases.

File: test/module/index.js (hunk 1 above)
Comment (paulirish) [inline test/module/index.js]: This test only asserts there was truthy output. Unless it's looking for something more specific, I don't think we need to run everything. That's technically what the smoketest does.

**Comment by paulirish** [comment]
> : ideally we should take a list of tech that we want to test (default: all), and we load the appropriate gatherers and set up the scheduler to only run the required phases.

agree, but It feels like that is a few weeks out. brendan is looking at it but will be in polish mode in the short term.

i had a few devtools engineers running LH this week, and this hack was pretty neccessary for iterating quickly. well. and same me while developing.

**Comment by paulirish** [comment]
closing as we're getting closer on audit whitelisting.

**Pull request closed without merging**